### PR TITLE
Note what each crate is for in its readme

### LINKED ISSUE #493: What is it you DO here?
Add some documentation of what each crate does and what its name means would be grand. Some things are self explanatory - ufo2fontir for example - but others aren't, fontdrasil is still a bit of a mystery, and this then leads to being uncertain as to whether or not it's okay to use types from that crate in other ones, that sort of thing. A tiny README in each crate with three sentences this is what it does and this is where you should/shouldn't use it would make the world of difference

### PR DESCRIPTION
**Pull request by rsheeter**
Fixes #493. JMM if happy.

### CODE REVIEW

File: fontir/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# fontir
+
+[![MIT/Apache 2.0](https://img.shields.io/badge/license-MIT%2FApache-blue.svg)](#license)
+
+fontir: Font Internal Representation
+
+This crate defines the internal representation (IR) of a font used by fontc and common operations
+on that IR.
+
+It should be referenced by fontc and any frontend (_format_2ir) crates.
```
Comment (anthrotype) [inline fontir/README.md:10]:
```suggestion
It should be referenced by fontc and any frontend (_format_2fontir) crates.
```

File: fontc/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+[![MIT/Apache 2.0](https://img.shields.io/badge/license-MIT%2FApache-blue.svg)](#license)
+
+fontc: Font Compiler
+
+This crate provides the command line entrypoint into our font compiler.
+
+It's primary task is to accept arguments from the user, create a task graph, and execute it.
+
+This crate can reference all other crates in the workspace.
```
Comment (anthrotype) [inline fontc/README.md:11]: how about a DOT graph that visualizes all the interdependencies?

**Review by anthrotype — APPROVED** [review]

File: fontdrasil/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# fontdrasil
+
+[![MIT/Apache 2.0](https://img.shields.io/badge/license-MIT%2FApache-blue.svg)](#license)
+
+fontdrasil: Connects all layers of fontc. A reference to Yggdrasil, the world tree.
```
Comment (cmyr) [inline fontdrasil/README.md:5]:
```suggestion
fontdrasil: Common types and functionality shared between all layers of fontc.
```

File: fontir/README.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# fontir
+
+[![MIT/Apache 2.0](https://img.shields.io/badge/license-MIT%2FApache-blue.svg)](#license)
+
+fontir: Font Internal Representation
```
[the remark this answers is not in the document]
Comment (anthrotype) [reply fontir/README.md:5]: me too

File: fontdrasil/README.md (hunk 3 above)
Comment (rsheeter) [reply fontdrasil/README.md:5]: Less Yggdrasil is clearly worse. I'll take your preamble but leave the Yggdrasil.

File: fontir/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (rsheeter) [reply fontir/README.md:5]: it is, editing sleepy ftw. Will fix.

File: fontc/README.md (hunk 2 above)
Comment (rsheeter) [reply fontc/README.md:11]: I think perhaps that would be good in the root readme as a follow-on?

Comment (rsheeter) [reply fontc/README.md:11]: https://github.com/googlefonts/fontc/issues/607

**Pull request merged**